src/parser/Expression.ts

```diff
@@ -291,7 +291,7 @@ export class FunctionParameterExpression extends Expression {
             result.push(' ');
             result.push(new SourceNode(this.asToken.range.start.line + 1, this.asToken.range.start.character, state.pathAbsolute, 'as'));
             result.push(' ');
-            result.push(new SourceNode(this.typeToken.range.start.line + 1, this.typeToken.range.start.character, state.pathAbsolute, this.typeToken.text));
+            result.push(util.bscTypeToTypeString(this.type));
```

Comment:
Still need a `SourceNode`. See this comment: https://github.com/rokucommunity/brighterscript/pull/262/files#r551867868